integrations/pennylane/syncs/suppliers.ts

```diff
@@ -0,0 +1,32 @@
+import type { CreateSupplier, NangoSync, PennylaneSupplier, ProxyConfiguration } from '../../models.js';
+import { toSupplier } from '../helpers.js';
+
+export default async function fetchData(nango: NangoSync) {
+    const config: ProxyConfiguration = {
+        // https://pennylane.readme.io/reference/suppliers-post
```

Comment:
Documentation should be to https://pennylane.readme.io/reference/suppliers-get

Reply:
this has been updated